src/components/DatePicker/DatePicker.stories.tsx

```diff
@@ -0,0 +1,84 @@
+import { ArgsTable, Description, Stories, Title } from "@storybook/addon-docs";
+import { Story } from "@storybook/react/types-6-0";
+import moment from "moment";
+import React from "react";
+import DatePicker from "./";
+
+export default {
+  title: "Components/Inputs/DatePicker",
+  component: DatePicker,
+  parameters: {
+    docs: {
+      source: { type: "code" },
+      page: () => (
+        <>
+          <Title />
+          <Description
+            markdown={[
+              "The wrapper of [react-dates](https://github.com/airbnb/react-dates).",
+              "",
+              "For more detail props, please see [it](https://github.com/airbnb/react-dates#singledatepicker).",
+            ].join("\n")}
+          />
+          <ArgsTable of={DatePicker} />
+          <Description
+            markdown={[
+              "## When the display is strange",
+              "",
+              "```tsx",
+              "// Please import css from `react-dates`.",
+              'import "react-dates/lib/css/_datepicker.css";',
+              "```",
```

Comment:
```suggestion
              "```typescript",
              "",
              'import "react-dates/lib/css/_datepicker.css";',
              "```",
```

Reply:
これでいい感じになると思う
![image](https://user-images.githubusercontent.com/50351271/191929053-0493979c-c303-4ec3-945d-d068ac6e034c.png)

Reply:
`Please import css from `react-dates`.` を追加するとしたらこうなるか

```tsx
markdown={[
  "## When the display is strange",
  "",
  "",
  "",
  "```tsx",
  "",
   'import "react-dates/lib/css/_datepicker.css";',
  "```",
]}
```

Reply:
ありがとうございます 🎉